pom.xml

```diff
@@ -45,15 +45,16 @@
     <!-- XXX If you change these versions please ensure the bundled LICENSE/NOTICE in
          hbase-oss is correct
      -->
+    <assertj.version>3.12.2</assertj.version>
     <audience-annotations.version>0.5.0</audience-annotations.version>
     <aws-java-sdk.version>1.11.1026</aws-java-sdk.version>
     <commons-io.version>2.5</commons-io.version>
     <commons-lang3.version>3.6</commons-lang3.version>
     <curator.version>4.2.0</curator.version>
     <enforcer.version>3.0.0-M3</enforcer.version>
     <extra.enforcer.version>1.2</extra.enforcer.version>
-    <hadoop2.version>2.9.2</hadoop2.version>
-    <hadoop3.version>3.2.2</hadoop3.version>
+    <hadoop32.version>3.2.2</hadoop32.version>
+    <hadoop33.version>3.3.1</hadoop33.version>
     <hbase1.version>1.4.10</hbase1.version>
     <hbase2.version>2.3.6</hbase2.version>
```

Comment:
Should we remove the hbase1 profile with this change as hadoop2 support is being dropped?

Reply:
Yep, agreed. If hadoop2 is being dropped, then I think that implies hbase 1 also being dropped (as HBase 1 still required Hadoop 2 support, if memory serves).

Reply:
Yes, HBase 1 does not support Hadoop3 based on the Ref.Guide. https://hbase.apache.org/book.html#hadoop